source/declarations.tex

```diff
@@ -3578,7 +3578,7 @@
 A \grammarterm{member-declarator} with an explicit-object-parameter-declaration
 shall not include
 a \grammarterm{ref-qualifier} or a \grammarterm{cv-qualifier-seq} and
-shall not be declared \keyword{static} or \keyword{virtual}.
+shall not be declared \keyword{static}.
```

Comment:
Do we have the same problem with the implicit `static` for allocation functions?
```
struct A {
  // I'm not sure which parameter needs to be size_t, but
  // one of them does...
  void *operator new(this size_t a, size_t b);
};
```

Reply:
See separate commit.